rehive/api/resources/admin_resources.py

```diff
@@ -184,9 +204,10 @@ def get_resource_name(cls):
 
 class APIAdminTransactions(ResourceList, ResourceCollection):
     def __init__(self, client, endpoint, filters=None):
-        self.resources = {
-            APIAdminWebhooks
-        }
+        self.resources = (
+            APIAdminWebhooks,
```

Comment:
I think this `APIAdminWebhooks` should not be here. As far as I know there is no webhook sub-resource on transactions.